vzstorage-pd/virtuozzo-provisioner.go

```diff
@@ -83,15 +188,25 @@ func (p *vzFSProvisioner) Provision(options controller.VolumeOptions) (*v1.Persi
 
 	glog.Infof("Add %s %s", share, capacity.Value())
 
-	ploop_options := map[string]string{}
-	for k,v := range options.Parameters {
-		ploop_options[k] = v
+	storage_class_options := map[string]string{}
+	for k, v := range options.Parameters {
+		storage_class_options[k] = v
 	}
 
-	ploop_options["volumeId"] = share
-	ploop_options["size"] = fmt.Sprintf("%d", bytes)
+	storage_class_options["volumeId"] = share
+	storage_class_options["size"] = fmt.Sprintf("%d", z)
 
-	if err := volume.Create(ploop_options); err != nil {
+	secret, err := p.client.Core().Secrets(storage_class_options["secretNamespace"]).Get(storage_class_options["secretName"])
+	if err != nil {
+		return nil, err
+	}
+
+	if err := initVstorage(storage_class_options, secret.Data); err != nil {
```

Comment:
I think we don't need to mount a vstorage each time. Pls, talk with @dmission, he is solving a similar problem for ploop-flexvol